services/core/shared/core.js

```diff
@@ -329,6 +329,36 @@ const getBlocks = async params => {
 const setReadyStatus = status => { readyStatus = status; };
 const getReadyStatus = () => readyStatus;
 
+const calcExpDecay = (NUM_BLOCKS, DECAY_RATE) => {
+	const expDecay = (1 - Math.pow(1 - DECAY_RATE, 1 / NUM_BLOCKS)).toFixed(5);
+	return expDecay;
+};
+
+const EMAcalc = async (feePerByte, prevFeeEstPerByte) => {
```

Comment:
`EMAcalc` can be a synchronous method.

Reply:
Now `EMAcalc` is a synchronous function.